packages/react-docs/patternfly-docs/patternfly-docs.source.js

```diff
@@ -44,9 +44,10 @@ module.exports = (baseSourceMD, sourceProps) => {
   sourceMD(path.join(reactCorePath, '/**/demos/**/*.md'), 'react-demos');
 
   // React-table MD
-  sourceMD(path.join(reactTablePath, '/**/TableComposable/examples/*.md'), 'react-composable');
-  sourceMD(path.join(reactTablePath, '/**/demos/*.md'), 'react-demos');
+  sourceMD(path.join(reactTablePath, '/**/TableComposable/examples/*.md'), 'react');
   sourceMD(path.join(reactTablePath, '/**/Table/examples/*.md'), 'react-legacy');
+  sourceMD(path.join(reactTablePath, '/deprecated/components/**/examples/*.md'), 'react-deprecated');
```

Comment:
Unfortunately, this will require some changes in `documentation-framework` to work properly. According to the issue this PR is linked to, it looks like the expectation is for there to be a `React deprecated` tab, which currently the doc framework does not account for.

`react-next` is a keyword leveraged within the doc framework to distinguish between identically named components. The same will have to be done for `deprecated`.
https://github.com/patternfly/patternfly-org/pull/3126

If this line is to be included in this PR, I would expect changes in patternfly-org's doc framework to be added/merged first.

Reply:
I can get a PR up for that.

Reply:
https://github.com/patternfly/patternfly-org/pull/3291

Reply:
So the build in @nicolethoen's PR will continue to fail because the path for `/deprecated` is not setup yet (other parts of this PR). So I would suggest commenting this line out with a TODO (maybe paste the PR link above in that comment) or removing this line altogether until https://github.com/patternfly/patternfly-org/pull/3291 is reviewed and merged.

Reply:
Thank you for clarifying @jeffpuzzo - that sounds like a good plan. Updated to reflect your suggestion.